### ISSUE TITLE
Text edit keyboard shortucts on non-english layouts

### ISSUE THREAD

**Issue by pk-pavlk** [vnotex/vnote#928]
OS Version: Linux (have tested on Windows as well)
VNote Version: 2.7

Symptoms:
Using text editing shortcuts for headings doesn't work with some non-english layouts (I have tested it on Czech (standard QWERTZ) and French layouts).

Pressing Ctrl-2 (3,4..) does nothing. In Czech layout Ctrl-1 increases font size (1 key without modifier is '+' ). In French layout, Ctrl-4 invokes the "Insert image" function (4 key is apostrophe). 

I have not been able to use the layout remapping feature to counter this, it behaves the same even if remapping is set.

How to Repro:
Try pressing Ctrl-1 (2,3..) with Czech or French layout in text edit mode.

Thanks for looking into it.

**Comment by tamlok** [comment]
The key layout mapping currently is used maily for Captain mode. Will check if it is applicable to Vim mode. Could you check if Captain mode is working with a non-English layout being set?

**Comment by pavlkx** [comment]
Yes, captain mode is working properly when the remapping is set.

Just to clarify, I'm not talking about Vim mode, but about normal text editing. Example of function that doesn't work (from documentation):

> Ctrl+\<Num\>
> Insert title at level <Num>. <Num> should be 1 to 6. Current selected text will be changed to title > if exists.
